examples/prisma-smart-subscriptions-apollo/prisma/schema.prisma

```diff
@@ -0,0 +1,41 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+generator client {
+  provider = "prisma-client-js"
```

Comment:
this looks to be missing the output location.  In a mono-repo we can't generate the client into the default location

Reply:
Just fixed e.e